teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(name = 'Unknown', age = nil, specialization:, parent_permission: true)
+    super(name, age, parent_permission: parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] As I mentioned in the previous comment, I think it would  be better if you correct the typo error in your constructor method, `specialization:`, right now your app is crushing by showing that the `initialize` method has wrong number of arguments (given 3, expected 0..2; required keyword: classroom) (`ArgumentError`).
You can correct it you following the same example like the one in the previous comment.